docs/specs/code/COIN_PRICE.md

```diff
@@ -0,0 +1,161 @@
+# Abstract
+
+Describe the ability to see and input ADA amount in another currency.
+
+# Proposal
+
+This feature includes two aspects:
+
+## 1. Input
+When the UI contains an ADA amount input, it should let the user input the number in another currency and covert it to ADA amount at the present conversion rate.
+
+## 2. Output
+When the UI shows an ADA amount, additionally it should be able to show the amount in another currency. If the ADA amount is related to an existing transaction, the user will be able to see the price at two different moments:
+-right now
+-at the moment of the transaction.
+
+The following fiat currencies will be supported:
+
+- USD	US dollar	
```

Comment:
Do we really want to track this many currencies? It leads to a higher storage requirement to store the historical value. Probably everybody knows at least one major currency so probably we can shorted this list.

Reply:
I would just track top 5 at the most: USD, JPY, EUR, CNY and KRW